build.sbt

```diff
@@ -1,38 +1,42 @@
+import com.jsuereth.sbtpgp.PgpKeys.publishSigned
 import sbt.Keys._
 import sbt._
 import xerial.sbt.Sonatype._
 
-
-lazy val commonSettings: Seq[Setting[_]]  = Seq(
-  scalaVersion := "2.13.0",
-  scalacOptions ++= Seq("-feature", "-unchecked"),
-  organization := "net.team2xh",
-  publishTo := sonatypePublishTo.value,
-  publishMavenStyle := true,
-  sonatypeProfileName := "net.team2xh",
-  sonatypeProjectHosting := Some(GitHubHosting(user="Tenchi2xh", repository="Scurses", email="tenchi@team2xh.net")),
-  developers := List(
-    Developer(id = "tenchi", name = "Hamza Haiken", email = "tenchi@team2xh.net", url = url("http://tenchi.me"))
-  ),
-  licenses := Seq("MIT" -> url("https://github.com/Tenchi2xh/Scurses/blob/master/LICENSE"))
+ThisBuild / scalaVersion := "2.13.6"
+ThisBuild / scalacOptions ++= Seq("-feature", "-unchecked")
+ThisBuild / organization := "net.team2xh"
+publishTo := sonatypePublishTo.value
+publishMavenStyle := true
```

Comment:
`publishTo` and `publishMavenStyle` are global settings and not common settings (sbt actually warns about this).